jest.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   testPathIgnorePatterns: ['src/integration'],
   testTimeout: 100000,
   verbose: true,
-  detectOpenHandles: true,
+  detectOpenHandles: false,
```

Comment:
I started getting an error related to TLSWRAP after adding some more integration tests. I should follow up on this later, but for now I don't want to block on it.